sunpycube/cube/NDCube.py

```diff
@@ -10,8 +12,9 @@
 import copy
 
 DimensionPair = namedtuple('DimensionPair', 'lengths axis_types')
+NDCubeSequencePair = namedtuple('NDCubeSequencePair', 'Size NDCubeDimensionPair')
```

Comment:
I think this should look more like the NDCube.dimensions.  So like
```SequenceDimensionPair = namedtuple('SequenceDimensionPair', 'shape axis_types')```
where ```NDCubeSequence.dimensions.shape``` is ```tuple([len(self.data)]+list(self.data[0].dimensions.lengths))```  and
```NDCubeSequence.dimensions.shape``` is ```tuple(["Sequence Axis"]+list(self.data[0].dimensions.axis_types))```